src/main/java/dude/task/Task.java

```diff
@@ -0,0 +1,63 @@
+package dude.task;
+
+/**
+ * Represents an abstract task for Dude.
+ * This abstract class provides the basic structure and behavior for various types of tasks.
+ */
+public abstract class Task {
+    protected String description;
+    protected boolean isDone;
```

Comment:
I think it is really good that your booleans follow the coding standard. i.e starts with "is"